tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/BootstrapTasks/GetBuildTools.cs

```diff
@@ -311,6 +312,7 @@ public override bool Execute()
                     {
                         copyFrom = Path.Combine(RemoteCopyFromRootDir, fl);
                         copyTo = Path.Combine(LocalBranchCopyToRootDir, fl);
+                        _buildToolsLogger.LogDebugInfo(string.Format("Copying {0} to {1}", copyFrom, copyTo));
```

Comment:
@dsgouda use the actual property name, do not use the field for BuildToolsLogger